gtclang/test/atlas-integration-test/reference/reference_diffusion.hpp

```diff
@@ -0,0 +1,63 @@
+#define GRIDTOOLS_CLANG_GENERATED 1
```

Comment:
what was the idea behind this? Would the goal not be a pure atlas diffusion as the reference?